template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Class-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Common"
+  ],
+  "name": "Class",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "Creates a new C# class",
+  "tags": {
+    "language": "C#",
+    "type": "item"
+  },
+  "groupIdentity": "DotNet.Common.Class",
+  "precedence": "0",
+  "identity": "DotNet.Common.Class.CSharp",
+  "shortName": "class",
+  "sourceName": "Class1",
+  "primaryOutputs": [
+    {
+      "path": "Class1.cs"
+    }
+  ],
+  "symbols": {
+    "DefaultNamespace": {
+      "type": "bind",
+      "binding": "msbuild:RootNamespace",
+      "replaces": "Company.ClassLibrary1"
+    },
+    "NETCoreSdkVersion": {
+      "type": "bind",
+      "binding": "msbuild:NETCoreSdkVersion"
+    },
+    "ImplicitUsings": {
+      "type": "bind",
+      "binding": "msbuild:ImplicitUsings"
+    },
+    "net6orLater": {
+      "type": "generated",
+      "generator": "regexMatch",
+      "datatype": "bool",
+      "parameters": {
+        "pattern": "^([6-9]\\.0\\.*)",
+        "source": "NETCoreSdkVersion"
+        }
+      },
```

Comment:
This should depend on TargetFramework, not SDK version. Same applies for other templates.